src/transaction/log_manager.c

```diff
@@ -7260,6 +7282,20 @@ log_dump_record_supplemental_info (THREAD_ENTRY * thread_p, FILE * out_fp, LOG_L
   return log_page_p;
 }
 
+static LOG_PAGE *
+log_dump_record_schema_modification_lock (THREAD_ENTRY * thread_p, FILE * out_fp, LOG_LSA * log_lsa,
+					  LOG_PAGE * log_page_p)
+{
+  LOG_REC_SCHEMA_MODIFICATION_LOCK *log_rec;
+
+  /* Get the DATA HEADER */
+  LOG_READ_ADVANCE_WHEN_DOESNT_FIT (thread_p, sizeof (*log_rec), log_lsa, log_page_p);
+  log_rec = ((LOG_REC_SCHEMA_MODIFICATION_LOCK *) ((char *) log_page_p->area + log_lsa->offset));
+  fprintf (out_fp, ", CLASSOID = %d|%d|%d\n", OID_AS_ARGS (&log_rec->classoid));
```

Comment:
Nit.
From experience, there's way too much uppercase in the log dump :)
```suggestion
  fprintf (out_fp, ", classoid = %d|%d|%d\n", OID_AS_ARGS (&log_rec->classoid));
```

Some values are good to be uppoercase:
```
LSA = 105818|1248, Forw log = 105818|1368, Backw log = 105818|1176,
     Trid =  -3, Prev tran logrec = 105818|776
     Type = LOG_SYSOP_START_POSTPONE,
     lastparent_lsa = -1|-1, prv_topresult_lsa = -1|-1, type = LOG_SYSOP_END_COMMIT 
     posp_lsa = 105818|776 
```
but not the labels

Reply:
OK, I agree.